src/scripts/embed/index.js

```diff
@@ -0,0 +1,43 @@
+/* @flow */
+
```

Comment:
This is the separate embed FE app gets loaded into the browser on requests to `/embed/:id` routes.